Containerfile

```
@@ -0,0 +1,15 @@
+FROM docker.io/library/alpine:3.14.2 AS build
+RUN apk add --no-cache gcc make musl-dev cmake pkgconfig linux-headers \
+      luajit-dev sqlite-dev zlib-dev brotli-dev zstd-dev
+COPY . /lwan
+WORKDIR /lwan/build
+RUN cmake .. -DCMAKE_BUILD_TYPE=Release
+RUN make -j
+
+FROM docker.io/library/alpine:3.14.2
+RUN apk add --no-cache luajit sqlite zlib brotli zstd-dev
```

Is there a non-dev version of zstd in Alpine that could be used instead?

```suggestion
RUN apk add --no-cache luajit sqlite zlib brotli zstd
```

---

Unfortunately on Alpine, libzstd.so is only present in zstd-dev and not zstd.